nautobot/extras/jobs.py

```diff
@@ -1040,12 +1041,18 @@ class DatabaseFileField(forms.FileField):
     widget = DBClearableFileInput
 
 
+class BootstrapStyleFileField(forms.FileField):
+    """File picker with UX bootstrap style and clearable checkbox."""
+
+    widget = ClearableFileInput
+
+
 class FileVar(ScriptVariable):
     """
     An uploaded file.
     """
 
-    form_field = DatabaseFileField
+    form_field = BootstrapStyleFileField
```

Comment:
I'm worried about this - we were already using a custom form widget here, though i'm not entirely sure of the history. Can you confirm locally that jobs with file inputs still can successfully submit the job via the UI and the job can still successfully access the file when run?

Reply:
It's working good locally (see picture below). If you want me to make sure that test is included in selenium I'll need to do some more tweaks to get volume mounting working to the selenium container so I can upload the file accurately. I suspect I can refactor how we are doing selenium in CI because right now its differently launched than locally which in my mind is problematic for maintenance. Should have CI run exactly the same invoke command that we run locally ideally.

![Screen Recording 2025-02-06 at 9 18 25 AM (1)](https://github.com/user-attachments/assets/3fbdc4ad-50cc-4e16-bda2-b06c3ad69c46)